content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,113 @@
-======================================
-Taking stock from different warehouses
-======================================
+===========================================================
+Sell stock from multiple warehouses using virtual locations
+===========================================================
 
-When you plan to deliver a customer, you don’t know in advance if the products will come from
-Warehouse A or Warehouse B. You may, in some cases, need to take stock from different warehouses.
-With *Odoo*, you can configure this by using the concept of virtual warehouses. Let us show you how
-to set those virtual warehouses.
+While keeping stock (and selling inventory) from one warehouse might work for smaller companies,
+bigger companies might need to keep stock in (and sell from) multiple warehouses in multiple
+locations.
```

Comment:
Can you explain why a company would use virtual locations? Is there an advantage? Why not use physical locations?

Reply:
I created a `.. spoiler::` tag to dive into this!